avocado/plugins/multiplex.py

```diff
@@ -48,7 +47,8 @@ def configure(self, parser):
 
         parser.add_argument('--filter-out', nargs='*', default=[],
                             help='Filter out path(s) from multiplexing')
-        parser.add_argument('--system-wide', action='store_true',
+        parser.add_argument('--system-wide', action='store_false',
```

Comment:
yes and eventually we'd have to also do it. But so far it makes sense to have it here and in the future we'd have to think about ways to enable single parsers (eg. if one wants to see only `multiplex` file, or only `foo-bar` file results). The current mux API does not cover this, but my initial thoughts were to use `data_inject` and `data_merge` with `name` in it and the actual merging/injecting would only happen during the multiplexation. This the `multiplexer` could keep only certain merges and remove the rest of them.

Basically the problem is this:

1. yaml parser is a plugin, Mux has no knowledge about it
2. Mux combines all `data_merge`s into a single tree
3. `avocado multiplex` should allow one to see results of partial results, like to see only results of one yaml file, without the system-wide options

But the (3) is hard because of (1). My proposal above was to use `data_merge(name, ...)` where name would be `mux-inject`, `mux-yaml` or `avocado-virt`. Then we could support filtering of those in `avocado multiplex`, so one can see only product of `mux-yaml` or his own `foo-bar` plugin.

Anyway I think we can live with the current solution for now. Or do you think I should include this? Or do you have other idea how to deal with this? The drawback of this solution is that it'd consume a bit more memory before the multiplexation as instead of storing the resulting tree we'd have to store all trees/datas in dictionaries before the multiplexation.